src/popup/components/OutOfFunds.tsx

```diff
@@ -0,0 +1,84 @@
+import React from 'react'
```

Comment:
This will be reworked in https://github.com/interledger/web-monetization-extension/issues/377 next.

Reply:
But I've added the boilerplate and integration.

Reply:
Note that solution for #377 will be added after this PR is merged. But feel free to ignore the UI part in this PR.